runtime/browser/runtime.cc

```diff
@@ -356,7 +356,11 @@ bool Runtime::CheckMediaAccessPermission(
 #else
   // This function may be called for a media request coming from
   // from WebRTC/mediaDevices. These requests can't be made from HTTP.
-  if (security_origin.SchemeIs(url::kHttpScheme))
+  std::string host = security_origin.host();
+  bool is_localhost = host.compare("localhost") ||
```

Comment:
don't we have a place like "utils" to put a function there so that this code is not duplicated?